Pidaq/Lib/wind_tunnel/air_properties.py

```diff
@@ -0,0 +1,116 @@
+"""Contains class that defines linear interpolation functions for wind tunnel testing
+
+Classes:
+    LinearInterpolation
+"""
+
+import csv
+import logging
+import sys
+from scipy import interpolate
+
+class LinearInterpolation:
+    """Creates linear interpolation functions for density and dynamic viscosity
+
+    This class defines two functions that are used to linearly interpolate
+    density (kg/m3) and dynamic viscosity (Pa*S) based on temperature (°C)
+    for the purpose of wind tunnel testing.
+    
+    Attributes:
+        densityFunc
+        viscosityFunc
+
+    Methods:
+        interpolateDensity
+        interpolateViscosity
+    """
+
+    def __init__(self, path: str):
+        """
+        Reads a CSV file to define two functions for density and dynamic
+        viscosity that can be used with temperature.  The CSV should have same
+        structure as current one in Drive, where the units are in the second row.
+        
+        Parameters:
+            path - path to a CSV file of the air properties
+
+        Raises:
+            ValueError - A unit (hence the column for a property) was not found
+        """
+        self.logger = logging.getLogger()
+
+        try:
+            open(path) # Open the CSV file
+        except IOError as err:
+            self.logger.error(err)
+            
+        with open(path) as file:
+            csv_reader = csv.reader(file, delimiter =',')
+            next(csv_reader) # Skip the first row
+            tempCol, densityCol, viscosityCol = -1,-1,-1
+            row = next(csv_reader) # Read the second row that contains the units
+        
+            for index, cell in enumerate(row): # This loop determines the column index for 
+                    # temperature, density, and dynamic viscosity based on units
+
+                if cell.find('°C') != -1:
+                    tempCol = index
+
+                if cell.find('[kg/m3]') != -1:
+                    densityCol = index
+
+                if cell.find('[Pa*S]') != -1:
+                    viscosityCol = index
+
+            # if a unit was not found, raise an error
+            if tempCol == -1: 
+                raise ValueError('Temperature unit "°C" was not found.')
+                
+            if densityCol == -1: 
+                raise ValueError('Density unit "kg/m3" was not found.')
+                
+            if viscosityCol == -1:
+                raise ValueError('Dynamic viscosity unit "Pa*S" was not found.')
```

Comment:
Not too sure we want to raise an error here - could log an error and then return, allowing the calling application to handle the error. However, I wouldnt worry about this right now. We can always update this later and/or perform a try/catch on the call to this method.